### ISSUE TITLE
iceprod2 - global queueing (Trac #844)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#838]
There are several parts to global queueing:

* Site requests tasks from master
 * provides available resources at site

* Master responds with tasks to queue
 * This may contain dataset,job,task updates, or any other necessary information to run the task.

* Site must handle response, updating local DB

Depends on #837 (resource collection)

Migrated from https://code.icecube.wisc.edu/ticket/844
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:28", 
    "description": "There are several parts to global queueing:\n\n* Site requests tasks from master\n * provides available resources at site\n\n* Master responds with tasks to queue\n * This may contain dataset,job,task updates, or any other necessary information to run the task.\n\n* Site must handle response, updating local DB\n\nDepends on #843 (resource collection)\n", 
    "reporter": "david.schultz", 
    "cc": "ddelventhal", 
    "resolution": "fixed", 
    "_ts": "1550067088921308", 
    "component": "iceprod", 
    "summary": "iceprod2 - global queueing", 
    "priority": "critical", 
    "keywords": "", 
    "time": "2015-01-06T18:04:52", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150407T22:05:28: david.schultz commented:

> In changeset 131168:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="131168"
> mega commit:
> - the concept of a "resource" that tasks can use
>   TODO: not used by queueing yet.
> - global queueing framework
>   TODO: make sure local queueing and global queueing work together.
> - separate out dbmethods tests
> - fix up some failing tests
> 
> This partially addresses #837 and #838.
> ```

**Comment by icecube-trac** [comment]
Trac update at 20150504T15:57:24: david.schultz changed milestone from "" to "IceProd 2.0"

**Comment by icecube-trac** [comment]
Trac update at 20150831T19:06:19: david.schultz changed status from "new" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20151127T17:37:38: david.schultz changed priority from "major" to "critical"

**Comment by icecube-trac** [comment]
Trac update at 20151204T23:11:36: 
* david.schultz changed resolution from "" to "fixed"
* david.schultz changed status from "accepted" to "closed"
* david.schultz commented:

> In changeset 140118:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="140118"
> wrap up the global queueing changes.  I *think* it should work now. closes #838
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:28: 
* olivas changed milestone from "IceProd 2.0" to ""
* olivas commented:

> Milestone IceProd 2.0 deleted
